### ISSUE TITLE
Usage in Clojurescript?

### ISSUE THREAD

**Issue by joshcho** [merrickluo/lsp-tailwindcss#52]
**make sure to read the [troubleshooting](https://github.com/merrickluo/lsp-tailwindcss#troubleshooting) guide first.**

This is a bit of a shot in the dark, but is it possible to use lsp-tailwindcss in clojurescript? I tried add-on, but it doesn't work (as expected).

**Comment by merrickluo** [comment]
1. add the major mode use in clojurescript to `lsp-tailwindcss-major-modes`
2. make sure there is a `tailwind.config.js` somewhere

**Comment by zikajk** [comment]
@joshcho Were you able to solve this? I am trying to use this one with Eglot and CLJS...

**Comment by merrickluo** [comment]
@zikajk FYI you don't need this package for using with eglot, and you can't use two language servers in a single buffer with eglot too.

**Comment by zikajk** [comment]
@merrickluo Right. By "you don't need this package..." you mean there is some other way to get Eglot + Tailwind LSP working?

**Comment by merrickluo** [comment]
eglot works differently with lsp-mode, theoretically, if you only need to tell it how to start a language server, for example, if you want to use tailwindcss language server in `clojurescript-mode`.
```emacs-lisp
(add-to-list 'eglot-server-programs
                 '((clojurescript-mode :language-id "html") . ("tailwindcss-language-server")))
```

it may or may not work, since tailwindcss's language server might need some specific config. I'll probably try it out when eglot got multiple server support, cuz using it alone doesn't make much sense xD,

**Comment by dgutov** [comment]
Try to call `M-x lsp-workspace-folders-add` first.

**Comment by merrickluo** [comment]
close due to inactivity.
